app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  validates :name, presence: true, allow_blank: false
+  has_many :expenses, dependent: :destroy, foreign_key: :author_id
+  has_many :categories, dependent: :destroy, foreign_key: :author_id
```

Comment:
- [ ] Great job adding user name validation 👏🏻 Let's now validate all user input (email, password, ...) to make sure that anyone with bad intentions cannot compromise their app.